about.html

```diff
@@ -0,0 +1,132 @@
+    </div>
+    <section class="">
+        <div class="About">
+
+            <div class="About-text">
+
+                <h1>Welcome To Wills.</h1>
+                <p>Built in 2000 during the George William period, this hotel is located in the center of Kyengera, with
+                    easy access to the city's tourist attractions. It offers tastefully meals and clearly decorated
+                    rooms.
+                </p>
+            </div>
+            <div>
+                <ul class="About-list">
+                    <li><i class="fa-solid fa-circle-check"></i>
+                        <h2>20% Off On Accommodation.</h2>
+                    </li>
+                    <li><i class="fa-solid fa-circle-check"></i>
+                        <h2>Complimentary Daily Breakfast.</h2>
+                    </li>
+                    <li><i class="fa-solid fa-circle-check"></i>
+                        <h2>3 Pcs Laundry Per Day.</h2>
+                    </li>
+                    <li><i class="fa-solid fa-circle-check"></i>
+                        <h2>Free Wifi.</h2>
+                    </li>
+                    <li><i class="fa-solid fa-circle-check"></i>
+                        <h2>Discount 20% On Restaurants Services Travel & Camping</h2>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="work">
+        <h1>Discover Our Work</h1>
+        <p class="bg3">Built in 2000 during the George William period, this hotel is located in the center of Kyengera,
+            with
+            easy access to the city's tourist attractions. It offers tastefully meals and clearly decorated
+            rooms.</p>
+        <div class="image-div">
+
+            <div class=" bg1">
+
+                <h1 class="bg">Family Room</h1>
+            </div>
+            <div class=" bg2">
+
+                <h1 class="bg">Delux Room</h1>
+            </div>
+        </div>
+    </section>
+
+
+    <section class="sponser">
+        <h1>Our Partners</h1>
+        <hr>
+        <div class="sponser-image">
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/coinbase.svg" alt=""></a>
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/binance-Logo.png" alt=""></a>
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/Flow.png" alt=""></a>
+            <a href=""><img src="https://promzzykoncepts.github.io/NFT-Conference/images/moonpaymain.svg" alt=""></a>
+        </div>
+    </section>
+
+    <footer>
+
+        <div class="contact">
+            <h1>Wills Hotel</h1>
+        </div>
+        <p class="link">Copyright ©2022 All rights reserved | This is a Demo Website and all contacts provided are
+            null
+            and void.
+            This website is designed by <a href="kgeorgewilliam60@gmail.com" class="email">Kidde</a>
+        </p>
```

Comment:
- [ ] I think here you need to add width to the footer so you can align the logo and the copyrights in a horeziontal way so kindly fix them to be looks like the original design 🔥🔥🔥